book/src/07_parallel_join.md

```diff
@@ -0,0 +1,45 @@
+# [DRAFT] Parallel Join
+
+As mentioned in chapter dedicated on how to [dispatch][c3] systems
+Specs automatically parallises execution of systems that have
+non-conflicting component requirements (mutability XOR aliasing).
+
+What isn't automatically parallised are the joins made within single system:
+
+```rust,ignore
+    fn run(&mut self, (vel, mut pos): Self::SystemData) {
+        use specs::Join;
+        // This loop is ran sequentially on a single thread.
```

Comment:
I cannot tell if this sentence is correct.

Reply:
Should probably rather be "This loop **runs** sequentially on a single thread."

Reply:
Oh yes, sounds better.